tests/resources/test_snippets.py

```diff
@@ -320,6 +336,244 @@ def _compare_snippets(snippet_db, snippet_api):
         _compare_snippets(snippet_db, snippet_api)
 
 
+# a helper function of traversing of the list of snippets via API
+async def get_next_page(testapp, limit=3, marker=0):
```

Comment:
(minor) If it's a helper function, maybe prefix it with an underscore?

Reply:
Done!